### ISSUE TITLE
🔗 Unexpected Scrolling Behavior with popup and on the elementor website (Duplicate of #29175)

### ISSUE THREAD

**Issue by valentijnh** [elementor/elementor#29292]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

When a popup is triggered using Elementor on a mobile device (iOS) or a desktop Mac, the page scrolls all the way to the bottom unexpectedly. This behavior has been observed on multiple websites, including Elementor's own blog page.

### Steps to reproduce

1. Open [https://steelworks.nl/](https://steelworks.nl/) built with Elementor on an iOS device or a desktop Mac.
2. Wait a second or interact with the page.
3. Observe that the page scrolls unexpectedly to the bottom of the site.

Alternatively:

1. Visit the following Elementor blog page on an iOS device or a desktop Mac:  
   [https://elementor.com/blog/elementor-325-css-loading-anchor-offset-search-pagination/?lid=6751i55h0eu0&utm_source=braze&utm_medium=email&utm_campaign=3.25%20Core%20Release%20Pro%20&%20Hosting&utm_term=gif](https://elementor.com/blog/elementor-325-css-loading-anchor-offset-search-pagination/?lid=6751i55h0eu0&utm_source=braze&utm_medium=email&utm_campaign=3.25%20Core%20Release%20Pro%20&%20Hosting&utm_term=gif)
2. Scroll down to under the video.
3. Observe that the page scrolls all the way to the bottom unexpectedly.

### Expected behavior

The popup should appear without affecting the page's scroll position. Similarly, scrolling on the referenced page should not cause the page to jump to the bottom.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.11.9-MariaDB-cll-lve
	PHP Version: 8.2.25
	PHP Memory Limit: 512M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7
	Site URL: https://steelworks.nl
	Home URL: https://steelworks.nl
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: nl_NL
	Timezone: Europe/Amsterdam
	Debug Mode: Inactive

== Theme ==
	Name: Shoptimizer Child Theme
	Version: 1.2.2
	Author: CommerceGurus
	Child Theme: Yes
	Parent Theme Name: Shoptimizer
	Parent Theme Version: 2.7.9
	Parent Theme Author: CommerceGurus

== User ==
	Role: administrator
	WP Profile lang: nl-NL
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36

== Active Plugins ==
	3D FlipBook : Dflip Lite
		Version: 2.3.48
		Author: DearHive

	Advanced Dynamic Pricing for WooCommerce
		Version: 4.9.0
		Author: AlgolPlus

	All-in-One WP Migration
		Version: 7.87
		Author: ServMask

	Anti-Spam by CleanTalk
		Version: 6.44
		Author: СleanTalk - Anti-Spam Protection <welcome@cleantalk.org>

	Cart2Cart: Password Migration
		Version: 1.1
		Author: MagneticOne

	Classic Editor
		Version: 1.6.5
		Author: WordPress Contributors

	CommerceGurus CommerceKit
		Version: 2.2.9
		Author: CommerceGurus

	Cookie banner plugin for WordPress – Cookiebot CMP by Usercentrics
		Version: 4.3.10
		Author: Usercentrics A/S

	Custom Taxonomy Order
		Version: 4.0.1
		Author: Marcel Pol

	Documents for WooCommerce but better
		Version: 1232322332.0.3
		Author: Dragos Micu

	Elementor
		Version: 3.25.6
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.3
		Author: Elementor.com

	FluentSMTP
		Version: 2.2.81
		Author: FluentSMTP & WPManageNinja Team

	Hamerhaai
		Version: 0.0.52
		Author: Blub

	Health Check &amp; Troubleshooting
		Version: 1.7.1
		Author: The WordPress.org community

	JetSmartFilters
		Version: 3.5.7.1
		Author: Crocoblock

	Kadence WooCommerce Email Designer
		Version: 1.5.13
		Author: Kadence WP

	Kirki Customizer Framework
		Version: 5.1.0
		Author: Themeum

	Login as User
		Version: 1.5.6
		Author: Web357

	ManageWP - Worker
		Version: 4.9.20
		Author: GoDaddy

	Menu Image
		Version: 3.11
		Author: Freshlight Lab

	MultiSafepay
		Version: 6.6.2
		Author: MultiSafepay

	PDF Invoices &amp; Packing Slips for WooCommerce
		Version: 3.9.0
		Author: WP Overnight

	Perfmatters
		Version: 2.3.4
		Author: forgemedia

	Rank Math SEO
		Version: 1.0.232
		Author: Rank Math SEO

	Rank Math SEO PRO
		Version: 3.0.1
		Author: Rank Math

	Repeat Order for Woocommerce
		Version: 1.3.3
		Author: polyres

	Setary — Bulk Editor for WooCommerce Products
		Version: 1.13.0
		Author: Setary

	Smart Image Resize PRO
		Version: 1.8.4
		Author: Nabil Lemsieh

	WooCommerce
		Version: 9.4.1
		Author: Automattic

	WooCommerce NL Postcode Checker
		Version: 2.10.9
		Author: WP Overnight

	WP All Export - WooCommerce Export Add-On Pro
		Version: 1.0.9
		Author: Soflyy

	WP All Export Pro
		Version: 1.9.1
		Author: Soflyy

	WP All Import - User Import Add-On Pro
		Version: 1.1.8
		Author: Soflyy

	WP All Import - WooCommerce Import Add-On Pro
		Version: 4.0.0
		Author: Soflyy

	WP All Import Pro
		Version: 4.8.7
		Author: Soflyy

	WP Rollback
		Version: 2.0.7
		Author: WP Rollback

	xCore Rest API extension
		Version: 1.12.4
		Author: Dealer4Dealer

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

	Health Check Troubleshooting Mode
		Version: 1.9.2
		Author: 

	ManageWP - Worker Loader
		Version: 1.0.0
		Author: GoDaddy

	mu-setary-helper.php
		Version: 
		Author: 


== Eigenschappen ==
	Custom Fonts: 2
	Custom Icons: 0

== Integraties ==
	
	woocommerce: Actief


== Elementor experimenten ==
	Verbeterd laden van CSS: Standaard inactief
	Inline lettertype iconen: Standaard actief
	Extra aangepaste breakpoints: Standaard actief
	Container: Standaard actief
	Upgrade Swiper bibliotheek: Standaard actief
	Geneste elementen prestaties: Standaard actief
	Geoptimaliseerde laden van besturing: Standaard actief
	Geoptimaliseerde markup: Standaard inactief
	Swiper CSS bestanden conditioneel laden: Standaard inactief
	Plugin onboarding: Standaard actief
	CSS smooth scroll: Standaard actief
	Standaard naar nieuwe Thema bouwer: Standaard actief
	Elementor home scherm: Standaard actief
	Landingspagina&#039;s: Standaard actief
	Geneste elementen: Standaard actief
	Editor top balk: Standaard actief
	Pagina&#039;s paneel: Standaard inactief
	Element caching: Standaard inactief
	Link in bio: Standaard actief
	Zwevende knoppen: Standaard actief
	Elementor editor gebeurtenissen: Standaard inactief
	Atomic widgets: Standaard inactief
	Launchpad checklist: Standaard inactief
	Menu: Standaard inactief


== Log ==
	
Log: showing 20 of 402024-08-26 15:16:55 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.23.2',
  'to' => '3.23.3',
)]
2024-09-19 10:42:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.23.3',
  'to' => '3.24.2',
)]
2024-09-19 10:42:05 [info] elementor-pro::elementor_pro_updater Started 
2024-09-19 10:42:05 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-09-19 10:42:05 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-09-19 10:42:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.23.3',
  'to' => '3.24.2',
)]
2024-09-19 10:44:12 [info] elementor::elementor_updater Started 
2024-09-19 10:44:12 [info] Elementor/Upgrades - _on_each_version Start  
2024-09-19 10:44:12 [info] Elementor/Upgrades - _on_each_version Finished 
2024-09-19 10:44:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.23.4',
  'to' => '3.24.3',
)]
2024-11-14 14:20:37 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.3',
  'to' => '3.25.6',
)]
2024-11-14 14:20:37 [info] elementor::elementor_updater Started 
2024-11-14 14:20:37 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-14 14:20:37 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-14 14:20:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.3',
  'to' => '3.25.6',
)]
2024-11-14 14:38:30 [info] elementor-pro::elementor_pro_updater Started 
2024-11-14 14:38:30 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-11-14 14:38:30 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-11-14 14:38:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.2',
  'to' => '3.25.3',
)]
2024-11-14 14:38:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.2',
  'to' => '3.25.3',
)]

JS: showing 2 of 2JS: 2024-07-16 09:57:39 [error X 3][../wp-admin/load-scripts.php?c=0&amp;loadchunk_0=jquery-core,jquery-migrate,utils,underscore,backbone,wp-util&amp;ver=6.5.5:2:28760] Cannot read properties of undefined (reading &#039;value&#039;) 
JS: 2024-08-05 09:22:19 [error X 3][../wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.23.2:3:66072] Cannot convert undefined or null to object 

PHP: showing 1 of 1PHP: 2024-11-01 13:49:50 [warning X 1][../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/fonts-manager.php::262] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibel

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by frassatidesigns** [comment]
Same issue here

**Comment by portrayaloflife** [comment]
This is a duplicate #29175

**Comment by nicholaszein** [comment]
Hello @valentijnh!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #29175 

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
